src/cook-web/src/app/c/[[...id]]/Components/ChatUi/ListenModeRenderer.scss

```
@@ -413,13 +474,19 @@
   }
 
   .listen-slide-player-mobile .slide-player__controls {
-    --slide-player-mobile-control-count: 5 !important;
+    --slide-player-mobile-control-count: 6 !important;
     grid-template-columns: repeat(
       var(--slide-player-mobile-control-count, 4),
       minmax(0, 1fr)
     ) !important;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)